README.md

```diff
@@ -1,2 +1,56 @@
 # Greetings
-A site that displays: "Hello World"
+A website that displays the text"Hello World"
+
+https://user-images.githubusercontent.com/62938735/124670432-e5234580-deab-11eb-975d-c795c2384459.jpg
```

Comment:
[OPTIONAL] If you include an image in your repo it is very necessary to ensure that it actually displays and not the link. Otherwise, it is not mandatory to include images and you can thus remove it. If you would like to see how to set up an image in a repo please refer to this  [link](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md)

Reply:
Thanks, I will make the changes